File: src/codes/Users_Manual/gain/focus_0.py
~from pyautd3 import Intensity, Focus, FocusOption, Phase
~
~x = 1.0
~y = 0.0
~z = 0.0
Focus(
    pos=[x, y, z],
    option=FocusOption(
        intensity=Intensity.MAX,
        phase_offset=Phase.ZERO,
    ),
)

File: src/codes/Users_Manual/gain/focus_0.rs
# use autd3::prelude::*;
# fn main() {
# let x = 0.;
# let y = 0.;
# let z = 0.;
# let _ = 
Focus {
    pos: Point3::new(x, y, z),
    option: FocusOption {
        intensity: EmitIntensity::MAX,
        phase_offset: Phase::ZERO,
    },
};
# }